features/data_ingress_v1.feature

```diff
@@ -11,14 +11,20 @@ Feature: Data Ingress v1
     Given a temporary source directory of files
         | file                   |  fixture                           |
         | pipeline-config.json   |  pipeline_config_basic_valid.json  |
-        | data.xml               |  data_sdmx_valid_1.sdmx            |
+        | data.xml               |  data.xml                          |
     And v1_data_ingress starts using the temporary source directory
     Then the pipeline should generate no errors
+    And I read the csv output 'data.csv'
+    And the csv output should have '9744' rows
+    And the csv output has the columns
+          | Column 1 | Column 2 | Column 3 |
```

Comment:
Resolved but note: I made it check for the existence of some columns from the output, I didn't make it check all the columns because there is a very large number of them and I didn't want to put so many of them in the feature file table.